probe/kubernetes/reporter.go

```diff
@@ -0,0 +1,41 @@
+package kubernetes
+
+import "github.com/weaveworks/scope/report"
+
+// Reporter generate Reports containing Container and ContainerImage topologies
```

Comment:
I remember discussing why we chose this route vs. a new topology, but I don't remember the finer details :) Maybe you can add a bit more commentary?